src/components/GettingStarted.tsx

```diff
@@ -2,75 +2,98 @@ import React, { ReactElement } from 'react'
 import { StartButtons, SingleButtons } from './pager'
 
 const repoUrl: string = 'https://github.com/ustaxes/UsTaxes'
-const codeOfConductUrl: string = 'https://github.com/ustaxes/UsTaxes/blob/master/docs/CODE_OF_CONDUCT.md'
```

Comment:
I think this is a good example of long strings causing long lines and that's ok.